src/utils/validation.ts

```diff
@@ -45,12 +45,18 @@ export const beaconsSchema = z.record(beaconSchema);
 
 export const secretsSchema = z.object({ filename: z.string(), content: z.string() });
 
-export const airnodeDeploymentSchema = z.object({
+export const airnodeDeploymentSchemaAWS = z.object({
   config: airnodeConfigSchema,
   secrets: secretsSchema,
   aws: secretsSchema.optional(),
 });
 
+export const airnodeDeploymentSchemaGCP = z.object({
+  config: airnodeConfigSchema,
+  secrets: secretsSchema,
+  gcp: z.any().optional(),
```

Comment:
Do you mind briefly explaining why isn't this one also a `secretsSchema.optional()`?

Reply:
secretsSchema is meant for non json secrets. For jsons we can use a standard z object